dashboard/app/models/lesson.rb

```diff
@@ -168,7 +168,7 @@ def numbered_lesson?
   end
 
   def has_lesson_pdf?
-    return false if Unit.unit_in_category?('csf', script.name) && ['2017', '2018'].include?(script.version_year)
+    return false if script.csf? && ['2017', '2018'].include?(script.version_year)
```

Comment:
before making this change, I confirmed that all course offerings in category `csf`, which have any published units, already have marketing initiative `CSF`